src/form/Dropdown.tsx

```diff
@@ -7,7 +7,7 @@ interface MyProps {
   name: string,
   value: string,
   options: string[],
-  valueChanged: (e: React.ChangeEvent<HTMLInputElement>) => void,
+  valueChanged: (value: string | null) => void,
```

Comment:
This Dropdown.tsx does not require a review. This is an attempt to build the input dropdown but I have not finished it yet. I might come back to it later.